libfabric.vcxproj.filters

```diff
@@ -0,0 +1,332 @@
+      <Filter>Header Files\windows</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_abi.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_atom.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_file.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_lock.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_mem.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_osd.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_proto.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\fi_util.h">
+      <Filter>Header Files</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\poll.h">
+      <Filter>Header Files\windows</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\pthread.h">
+      <Filter>Header Files\windows</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\unistd.h">
+      <Filter>Header Files\windows</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\socket.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\time.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\ipc.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\mman.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\select.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\arpa\inet.h">
+      <Filter>Header Files\windows\arpa</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\net\if.h">
+      <Filter>Header Files\windows\net</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\netinet\in.h">
+      <Filter>Header Files\windows\netinet</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\netinet\ip.h">
+      <Filter>Header Files\windows\netinet</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\netinet\tcp.h">
+      <Filter>Header Files\windows\netinet</Filter>
+    </ClInclude>
+    <ClInclude Include="include\windows\sys\param.h">
+      <Filter>Header Files\windows\sys</Filter>
+    </ClInclude>
+    <ClInclude Include="prov\sockets\include\sock.h">
+      <Filter>Source Files\prov\sockets\include</Filter>
+    </ClInclude>
+    <ClInclude Include="prov\sockets\include\sock_util.h">
+      <Filter>Source Files\prov\sockets\include</Filter>
+    </ClInclude>
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="libfabric.def">
+      <Filter>Source Files</Filter>
+    </None>
+  </ItemGroup>
+</Project>
```

Comment:
Can you separate the project files into their own patch, so it's easier to review the source code changes?